tests/test_decorators.py

```diff
@@ -429,3 +429,40 @@ def strict_fields(self, data, original_data):
         assert 'baz' in errors
         assert len(errors['baz']) == 1
         assert errors['baz'][0] == 'Unknown field name'
+
+    def test_skip_on_field_errors(self):
+
+        class MySchema(Schema):
+            foo = fields.Int(required=True)
+            bar = fields.Int(required=True)
```

Comment:
Could you also test that the schema validators aren't run if a field validator (i.e. a validator callable passed as the `validate` argument) fails?